src/components/modals/ProgressModal.vue

```diff
@@ -122,3 +130,42 @@ export default {
   },
 };
 </script>
+
+<style lang="scss" scoped>
+.progress-form {
+  @media screen and (min-width: bp(s)) {
+    display: flex;
+  }
+
+  &__left {
+    display: flex;
+    flex-grow: 1;
+    flex-direction: column;
+    justify-content: space-between;
+    padding-right: 1rem;
+  }
+
+  ::v-deep input[name="value"] {
+    font-size: 1.5rem;
+  }
+  ::v-deep textarea[name="comment"] {
+    resize: vertical;
+    height: 195px;
```

Comment:
Tror vi kan komme unna å hardkode denne? Ved å flexe `progress-form` og la den `span`-en som kommentaren ligger inne i vokse.

Reply:
Må vel sette litt `height: 100%` nedover innenfor `span`en der om vi vil den skal matche, men kan prøve

Reply:
Ja. Måtte også inn med noe `calc`-greier da jeg testa for å trekke fra `margin`-ene.

Reply:
Litt av problemet er for øvrig at Flatpickr ikke er responsiv i det hele tatt, men alt er mulig med ekstra CSS!